### PR TITLE
License and README.md

### PR DESCRIPTION
**Pull request by bghszabi**
LICENSE file is updated according to official APIDevKit licensing by Graphisoft, and Readme is updated with APIDevKit and usage related info.

### CODE REVIEW

**Review by palenikm — APPROVED** [review]

**Review by kovacsv — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Archicad API Development Kit
 
-This repository contains the documentation of the [Archicad API](https://graphisoft.github.io/archicad-api-devkit/).
+This repository contains the [Development Kit](https://archicadapi.graphisoft.com/downloads), and the documentation of the [Archicad API](https://graphisoft.github.io/archicad-api-devkit/).
```
[the remark this answers is not in the document]
Comment (bghszabi) [reply README.md:3]: Jogos kérdés. Vehetjük ezentúl "single source of thruth"-nak ezt a repót APIDevKit szempontból, és akkor nem az apisite-ra mutatunk innen, hanem majd onnan fogunk ide, igaz?

[the remark this answers is not in the document]
Comment (akossomorjai) [reply README.md:3]: Igen, ez jó lenne. Egyetlen szempont mégis az ArchicadAPI site mellett, hogy ott tud csak regisztrálni (developer ID-t szerezni) → ezt egy extra sorral el lehetne intézni:
"To register as an Archicad API developer, please visit the [Archicad API developer site](https://archicadapi.graphisoft.com/)."

[the remark this answers is not in the document]
Comment (palenikm) [reply README.md:3]: This is the first topic of the API council meeting. If this is the single source of truth, then the API site's Download SDK page should be redirected here. One more question: where can a developer get previous versions of APIDevKit then?

[the remark this answers is not in the document]
Comment (bghszabi) [reply README.md:3]: As my knowledge, on the last API council meeting it was asked, that 25-26-27 versions of APIDevkit should be stored here in the github repository, and any version older than that can be accessed on the archicadapi site. In order to gain developer ID we should redirect the developer from here to the api site. We will mention it here, that any DevKit version up to 27 can be downloaded from there too.

So until the version 27, we should consider the apisite as the single source of truth, and version 28 can be released on GitHub which can be considered the official release, and the apisite link should redirect here.

[the remark this answers is not in the document]
Comment (akossomorjai) [reply README.md:3]: Yes, perfect, thanks!

**Review by akossomorjai — APPROVED** [review]

**Review by akomporday — COMMENTED** [review]
Thank you!

**Pull request merged**